.github/workflows/ci.yml

```
@@ -0,0 +1,64 @@
+      - name: Verify image
+        run: |
+          docker run --rm ${{ env.docker_image }}:temp --version
+  
+  build_binary:    
+    name: Build Binary
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Cache Cargo dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)